README.md

```diff
@@ -51,3 +57,17 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
 9. Untar pgsql, mongo data tar files. Restart required services.
 10. Update the Satellite default password for admin to `changeme`.
 11. Fix the katello assets link which point to an invalid path.
+
+
+## Changing the Hostname of a Satellite machine:
+1. Before you run the hostname change playbook, you have to make a manual change to the default capsule's hostname. Log into UI and rename the default capsule (under infrastructure->capsules) to the new hostname using both the name and url fields.
+2. Update the required sections in `roles/change_hostname/vars/main.yml`.
+3. Update the `inventory` file under the root of this project to add your Satellite VM's ip address.
+4. Run the ansible playbook:
+    ```console
+      # ansible-playbook -i inventory deploy-hostname-playbook.yml
+    ```
+   or if wanting to execute the playbook on localhost, specify the local IP address in the inventory file and run:
+    ```console
+      # ansible-playbook -i inventory -c local deploy-hostname-playbook.yml
+    ```
```

Comment:
Lines 70-73 are redundant.  You can very well specify the localhost ip address in inventory and run Line 68 without issues.